cartographer/mapping_2d/pose_graph.h

```diff
@@ -83,7 +83,7 @@ class PoseGraph : public mapping::PoseGraph {
       EXCLUDES(mutex_);
   void AddFixedFramePoseData(
       int trajectory_id,
-      const sensor::FixedFramePoseData& fixed_frame_pose_data);
+      const sensor::FixedFramePoseData& fixed_frame_pose_data) override;
```

Comment:
Shouldn't this be annotated `EXLUDES(mutex_)` here as well?

Reply:
Currently it's just a `LOG(FATAL)`, no mutex locking.

Reply:
I understand that it is not implemented, but `EXCLUDES(mutex_)` is part of the interface, i.e. you annotate that whoever calls this function must not have locked `mutex_`. It does not claim that it would be locked in the function. There is not reason to change the interface just because the function is currently not implemented.